i18n/en-US.properties

```diff
@@ -740,12 +742,14 @@ boxui.classification.classificationYellow = Yellow
 boxui.classification.definition = Definition
 # Button to edit classification on an item
 boxui.classification.edit = Edit
+# Text prepended to user or service that imported the classification.
+boxui.classification.importedFromText = Imported from
 # Default message for classification in the sidebar when there is none
 boxui.classification.missing = Not classified
 # Sentence stating which user or service modified a classification and on what date.
-boxui.classification.modifiedBy = {modifiedBy} on {modifiedAt}
+boxui.classification.modifiedBy = {modifiedActionLabel} {modifiedBy} on {modifiedAt}
```

Comment:
It's usually not recommended to combine message fragments that were translated independently, since translators need to consider the text as a whole and the order of the fragments might be different depending on the language. We can probably have a separate `boxui.classification.importedBy` message in order to avoid these issues.